content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,165 +1,320 @@
+To follow this tutorial, make sure that you have the following prerequisites in place:
 
-Click **Deploy**:
+- a [GitHub](https://github.com) account
+- a hosted PostgreSQL database and a connection string for it, e.g. `postgresql://username:password@your_postgres_db.cloud.com/db_identifier` (you can use Supabase, which offers a [free plan](https://dev.to/prisma/set-up-a-free-postgresql-database-on-supabase-to-use-with-prisma-3pk6))
+- a [Vercel](https://vercel.com) account
+- Node.js installed
 
-![Vercel deploy button](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/qqTHfji.png)
+If you [add a separate database for preview deployments](#3-optional-add-a-separate-database-for-preview-deployments), you will also need:
 
-Vercel prompts you to clone the repository:
+- a second hosted PostgreSQL database and a connection string for it
 
-![Vercel Clone repository](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/9iYnP6B.png)
-
-Choose a repository name to replace the pre-filled `prisma-vercel-deployment-example`. This is the name of the cloned repository in your GitHub account.
-
-> **Note:** If this is your first time using Vercel, you will be prompted to install the Vercel app in your GitHub account.
-
-Click **Create**. The configure project step opens:
+## Prisma workflow
 
-![Vercel configure project](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/FFZVPMm.png)
+At the core of Prisma is the [Prisma schema](/concepts/components/prisma-schema) – a file where you declaratively define your data model. The Prisma schema is a single source of truth for both Prisma Client and Prisma Migrate.
 
-Set the `DATABASE_URL` environment variable with the connection URL for your PostgreSQL database, which should look as follows: `postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__`
+In this guide, you will create the database schema with [Prisma Migrate](/concepts/components/prisma-migrate). Prisma Migrate takes your changes to the data model in your Prisma schema, generates a SQL migration file with the changes and applies it to your database.
 
-Finally, click **Deploy** to start the build:
+There are two main workflows for Prisma Migrate:
 
-![Vercel Build](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/zn7xFj9.png)
+- You can create and apply migrations during _local_ development with `prisma migrate dev`
+- You can apply the migrations to _production_ with `prisma migrate deploy`
 
-> **Note:** The build step defined in package.json as the `vercel-build` script, will run `prisma generate && prisma migrate deploy && next build` to generate Prisma Client, apply migrations, and build the Next.js project.
+For brevity, this guide does not cover how to create migrations with `prisma migrate dev`. Instead, it focuses on the production workflow with `prisma migrate deploy` and uses the Prisma schema and SQL migration that are included in the example code.
```

Comment:
That is an unexpected framing. Aren't usually the different workflows `db push` vs `migrate dev + deploy`?